tests/e2e-leg-9/verify-vrep/setup-vdb-target/base/setup-vdb.yaml

```diff
@@ -32,10 +32,28 @@ spec:
     - name: sc2
       size: 1
       type: secondary
+    - name: sc3
```

Comment:
You may need to split up the assertion for the create db step. There are a lot of things it needs to do, and it only has a 10 minute timeout. You may need to split the things to check over multiple steps so that it can complete within time. Each step has its own 10 minute timeout.